### PR TITLE
Update BulkImport.md with corrected variable name

### PR DESCRIPTION
**Pull request by StevenBorg**
## Description
Fixes variable name.

## Related issues
Addresses [issue #].

## Testing
Looking at MD file

## FHIR Team Checklist
- [ ] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with Azure API for FHIR if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
- [ ] Tag the PR with Azure Healthcare APIs if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

**Comment by StevenBorg** [comment]
I should have committed directly to main, because this documentation is going to be used in the AM tomorrow for training.  Oops!  Basically, configuring the FHIR OSS server requires an App Service setting and the name is incorrect in docs.

File: docs/BulkImport.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The FHIR server must have _init import mode_ set to _True_ for $import to work.
 FhirServer__Operations__Import__InitImportMode: True
 ```
 
-After the FHIR server app is ready, navigate to app service portal and click **Configuration**. Create the *FhirServer:Operations:Import:InitImportMode* setting if needed by clicking **New application setting**. Set the value to _True_ or _False_ as needed.
+After the FHIR server app is ready, navigate to app service portal and click **Configuration**. Create the *FhirServer:Operations:Import:InitialImportMode* setting if needed by clicking **New application setting**. Set the value to _True_ or _False_ as needed. (Note: the image below shows FhirServer:Operations:Import:*Init*ialImportMode, but the correct variable name is FhirServer:Operations:Import:*Initial*ImportMode.)
```
Comment (LTA-Thinking) [inline docs/BulkImport.md:59]: In the note you have InitialImportMode in both locations with different italics. Should the first one just be InitImportMode?

Comment (LTA-Thinking) [inline docs/BulkImport.md:59]: Should the two uses of the variable name in the paragraph above be updated with the correct variable name?

Comment (StevenBorg) [reply docs/BulkImport.md]: DOH!

Comment (StevenBorg) [reply docs/BulkImport.md]: I only found one, but did a quick pass.

Comment (LTA-Thinking) [reply docs/BulkImport.md]: The other one is the section title.

**Comment by LTA-Thinking** [comment]
These changes and others are also in #2364

**Comment by LTA-Thinking** [comment]
Work has been done in a different PR

**Pull request closed without merging**